ui.tests/test-module/specs/textinput/textinput.runtime.spec.js

```diff
@@ -86,4 +88,43 @@ describe("Form Runtime with Text Input", () => {
         cy.toggleDescriptionTooltip(bemBlock, 'tooltip_scenario_test');
     })
 
+    it("should show and hide other fields on a certain input", () => {
+        console.log(Object.entries(formContainer._fields))
+        const [textbox1, textBox1FieldView] = Object.entries(formContainer._fields)[0];
+        const [textbox2, textbox2FieldView] = Object.entries(formContainer._fields)[1];
+        const [textbox3, textbox3FieldView] = Object.entries(formContainer._fields)[2];
+        const input = "adobe";
+
+        cy.get(`#${textbox1}`).find("input").clear().type(input).blur().then(x => {
+            cy.get(`#${textbox3}`).should('be.visible')
+            cy.get(`#${textbox2}`).should('not.be.visible')
+        })
+    })
+
+    it("should enable and disable other textfields on a certain string input", () => {
+        const [textbox1, textBox1FieldView] = Object.entries(formContainer._fields)[0];
```

Comment:
on a general note I think we must not use any private members of models in our tests. this may change later on and they also might encourage our customers to do so which is not desired. We may have done it erraneously at few other places also but this needs to change.